
class pyFETestException(Exception):
    def __init__(self, *args, **kwargs):
        pass